[Merged by Bors] - feat(analysis/normed_space/multilinear): a few more bundled (bi)linear maps

urkud opened the pull request:
---
<!-- The text above the `---` will become the commit message when your
PR is merged. Please leave a blank newline before the `---`, otherwise
GitHub will format the text above it as a title.

Any other comments you want to keep out of the PR commit should go
below the `---`, and placed outside this HTML comment, or else they
will be invisible to reviewers.

If this PR depends on other PRs, please list them below this comment,
using the following format:
- [ ] depends on: #abc [optional extra text]
- [ ] depends on: #xyz [optional extra text]
-->

urkud committed:
feat(analysis/normed_space/multilinear): a few more bundled (bi)linear maps

eric-wieser commented on src/analysis/normed_space/multilinear.lean:
```diff
@@ -67,18 +67,19 @@ We use the following type variables in this file:
+universes u v v' wE wE₁ wE' wEi wG wG'
 variables {𝕜 : Type u} {ι : Type v} {ι' : Type v'} {n : ℕ}
-  {E : ι → Type wE} {E' : ι' → Type wE'} {Ei : fin n.succ → Type wEi}
+  {E E₁ : ι → Type wE} {E' : ι' → Type wE'} {Ei : fin n.succ → Type wEi}
```
```suggestion
  {E : ι → Type wE} {E₁ : ι → Type wE₁} {E' : ι' → Type wE'} {Ei : fin n.succ → Type wEi}
```

urkud committed:
Update src/analysis/normed_space/multilinear.lean

Thanks Eric! I added a `universe` to the list then forgot to put `E₁` into the new `universe`.

Co-authored-by: Eric Wieser <wieser.eric@gmail.com>

sgouezel commented on src/analysis/normed_space/multilinear.lean:
```diff
@@ -379,6 +374,30 @@ normed_group.of_core _ ⟨op_norm_zero_iff, op_norm_add_le, op_norm_neg⟩
 ⟨λ c f, f.op_norm_smul_le c⟩
 
+lemma op_norm_prod (f : continuous_multilinear_map 𝕜 E G) (g : continuous_multilinear_map 𝕜 E G') :
+  ∥f.prod g∥ = ∥(f, g)∥ :=
```
Is there a reason why you write `∥(f, g)∥` instead of `max (∥f∥) (∥g∥)` on the right? The latter would seem a little bit more straightforward to me, as there are many norms on the direct product and the reader might not be aware that the default one is the sup one in mathlib.

sgouezel reviewed (commented):
bors d+

urkud committed:
Explicitly write `max`

urkud committed:
Merge branch 'multilinear-bilinear' of git://github.com/leanprover-community/mathlib into multilinear-bilinear
